examples/medplum-mso-demo/src/data/core/sample-bundle.ts

```diff
@@ -0,0 +1,252 @@
+import { Bundle, Patient } from '@medplum/fhirtypes';
+
+// First bundle to create patients
+export const PATIENTS_BUNDLE: Bundle = {
+  resourceType: 'Bundle',
+  type: 'transaction',
+  entry: [
+    // Patient A
+    {
+      request: {
+        method: 'POST',
```

Comment:
Should make all of these "POST" requests into upserts, to make them idempotent:
https://www.medplum.com/docs/fhir-datastore/updating-data#upsert-operation